pkg/gen/misc.bzl

```diff
@@ -1,6 +1,31 @@
 # Generated by genbzl
 
 MISC_SRCS = [
+    "//pkg/cmd/mirror/npm:__cluster-ui.yarn.json",
```

Comment:
The effect of this is that `dev gen` will hoist all of these files into the `/pkg/cmd/mirror/npm` directory. I think that's not desired. Should be able to update the query for `misc` in `pkg/gen/genbzl/targets.go` to omit these files.

Reply:
Oh! Nope, that's not at all required — good catch.

Reply:
Fixed!